### PR TITLE
Ij locator markup

### PR DESCRIPTION
**Pull request by RomanIovlev**

### CODE REVIEW

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 1]
```diff
@@ -22,95 +23,95 @@
...
-    public static WebList $$(String locator, Object parent) {
+    public static WebList $$(@MarkupLocator String locator, Object parent) {
         return list(locator).setup(j->j.setParent(parent));
     }
-    public static WebList $$(By locator) {
+    public static WebList $$(@MarkupLocator By locator) {
         return list(locator);
     }
-    public static WebList $$(By locator, Object parent) {
+    public static WebList $$(@MarkupLocator By locator, Object parent) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:70]: Codacy Issue found: [The static method name '$$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766&bid=17736581)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 2]
```diff
@@ -22,95 +23,95 @@
...
     }
     public static WebList list(List<WebElement> els, String name) {
         return new WebList(els).setup(e->e.setName(name));
     }
-    public static WebList $$(String locator) {
+    public static WebList $$(@MarkupLocator String locator) {
         return list(locator);
     }
-    public static WebList $$(String locator, Object parent) {
+    public static WebList $$(@MarkupLocator String locator, Object parent) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:64]: Codacy Issue found: [The static method name '$$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 3]
```diff
@@ -22,95 +23,95 @@
...
         return list(locator).setup(j->j.setParent(parent));
     }
-    public static WebList $$(By locator) {
+    public static WebList $$(@MarkupLocator By locator) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:67]: Codacy Issue found: [The static method name '$$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 4]
```diff
@@ -22,95 +23,95 @@
...
-    public static UIElement element(By byLocator) {
+    public static UIElement element(@MarkupLocator By byLocator) {
         return new UIElement(byLocator);
     }
-    public static UIElement $(String locator) {
+    public static UIElement $(@MarkupLocator String locator) {
         return element(locator);
     }
-    public static UIElement $(String locator, Object parent) {
+    public static UIElement $(@MarkupLocator String locator, Object parent) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:37]: Codacy Issue found: [The static method name '$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 5]
```diff
@@ -22,95 +23,95 @@
...
         return locator.matches("[A-Z].*]")
             ? new UIElement().setName(locator)
             : element(defineLocator(locator));
     }
-    public static UIElement element(By byLocator) {
+    public static UIElement element(@MarkupLocator By byLocator) {
         return new UIElement(byLocator);
     }
-    public static UIElement $(String locator) {
+    public static UIElement $(@MarkupLocator String locator) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:34]: Codacy Issue found: [The static method name '$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 6]
```diff
@@ -22,95 +23,95 @@
...
     }
-    public static WebList list(By byLocator) {
+    public static WebList list(@MarkupLocator By byLocator) {
         return new WebList(byLocator);
     }
     public static WebList list(List<WebElement> els, String name) {
         return new WebList(els).setup(e->e.setName(name));
     }
-    public static WebList $$(String locator) {
+    public static WebList $$(@MarkupLocator String locator) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:61]: Codacy Issue found: [The static method name '$$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 7]
```diff
@@ -22,95 +23,95 @@
...
     }
-    public static UIElement $(By locator) {
+    public static UIElement $(@MarkupLocator By locator) {
         return element(locator);
     }
     public static UIElement $(WebElement webElement) {
         return new UIElement(webElement);
     }
-    public static UIElement $(By locator, Object parent) {
+    public static UIElement $(@MarkupLocator By locator, Object parent) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:46]: Codacy Issue found: [The static method name '$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java [hunk 8]
```diff
@@ -22,95 +23,95 @@
...
-    public static UIElement $(String locator) {
+    public static UIElement $(@MarkupLocator String locator) {
         return element(locator);
     }
-    public static UIElement $(String locator, Object parent) {
+    public static UIElement $(@MarkupLocator String locator, Object parent) {
         return element(locator).setup(j->j.setParent(parent));
     }
-    public static UIElement $(By locator) {
+    public static UIElement $(@MarkupLocator By locator) {
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java:40]: Codacy Issue found: [The static method name '$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5409766)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java [hunk 9]
```diff
@@ -121,10 +122,10 @@ public JDIBase setWebElements(List<WebElement> els) {
         return this;
     }
 
-    public JDIBase setLocator(String locator) {
+    public JDIBase setLocator(@MarkupLocator String locator) {
         return setLocator(defineLocator(locator));
     }
-    public JDIBase setLocator(By locator) {
+    public JDIBase setLocator(@MarkupLocator By locator) {
```
Comment (jreznot) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java:128]: `By` is not `String` so there is no need to mark it as locator.

**Pull request merged**